openmetadata-ui/src/main/resources/ui/src/pages/DashboardDetailsPage/DashboardDetailsPage.component.tsx

```diff
@@ -569,6 +570,34 @@ const DashboardDetailsPage = () => {
       });
   };
 
+  const deletePostHandler = (threadId: string, postId: string) => {
+    deletePostById(threadId, postId)
+      .then((res: AxiosResponse) => {
+        if (res.data) {
+          const { id } = res.data;
+          setEntityThread((pre) => {
+            return pre.map((thread) => {
+              const posts = thread.posts.filter((post) => post.id !== id);
+
+              return {
+                ...thread,
+                posts: posts,
+                postsCount: thread.postsCount - 1,
+              };
+            });
+          });
+          getEntityFeedCount();
+          showToast({
+            variant: 'success',
+            body: 'Post got deleted successfully',
```

Comment:
Here message should be, `Message deleted successfully` or `Reply deleted successfully`

Reply:
Take care of similar changes everywhere else